tools/aws-setup/setup-docker-swarm.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
It would be great to separate the actual ansible scripts from the shell scripts used in each machine as utils/final configs.